elements/rh-tile/rh-tile-group.ts

```diff
@@ -0,0 +1,184 @@
+  static readonly styles = [styles];
+
+  /**
+   * whether tile interaction is disabled but retains ability to be read by screen readers;
+   * preferred method of disabling instead of `disabled`
+   */
+  @property({ reflect: true, attribute: 'aria-disabled', type: String }) ariaDisabled = 'false';
+
+  /**
+   * if tile is checkable, whether only on tile can be checked
+   */
+  @property({ attribute: 'radio', type: Boolean }) radio = false;
+
+  /**
+   * Sets color theme based on parent context
+   */
+  @colorContextConsumer() private on?: ColorTheme;
+
+  /**
+   * Sets color palette, which affects the element's styles as well as descendants' color theme.
+   * Overrides parent color context.
+   * Your theme will influence these colors so check there first if you are seeing inconsistencies.
+   * See [CSS Custom Properties](#css-custom-properties) for default values
+   *
+   * Tile group always resets its context to `base`, unless explicitly provided with a `color-palette`.
+   */
+  @colorContextProvider()
+  @property({ reflect: true, attribute: 'color-palette' }) colorPalette?: ColorPalette;
+
+  #tiles: RhTile[] = [];
+  #initTiles = false;
+
+  #tabindex = new RovingTabindexController<HTMLElement>(this);
+  #internals = new InternalsController(this, { });
+
+  /**
+   * all slotted tiles
+   */
+  get tiles() {
+    return this.#tiles;
+  }
+
+  /**
+   * all selected tiles
+   */
+  get selected() {
+    const selected = this.#tiles?.filter(tile=> tile.checked);
+    const [first] = selected;
+    return this.radio ? first : selected;
+  }
+
+  protected firstUpdated(): void {
+    this.#internals.role = this.radio ? 'radiogroup' : null;
+    this.updateTiles();
+  }
+
+  protected updated(_changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>): void {
+    if (_changedProperties.has('radio')) {
+      this.#internals.role = this.radio ? 'radiogroup' : null;
+      let selected: RhTile | null | undefined;
+      this.#tiles.forEach(tile => {
+        if (tile.radio !== this.radio) {
+          tile.radio = this.radio;
+        }
+        if (this.radio && !selected && tile.checked) {
+          selected = tile;
+        }
+      });
+      this.selectTile(selected);
+    }
+    if (_changedProperties.has('ariaDisabled')) {
+      this.#tiles.forEach(tile => {
+        if (tile.ariaDisabled !== this.ariaDisabled) {
+          tile.ariaDisabled = this.ariaDisabled;
+        }
+      });
+    }
+  }
+
+  render() {
+    const { on = '', radio } = this;
+    return html`
+      <slot class="${classMap({ [on]: !!on, radio })}" @slotchange=${this.#onSlotchange} @select=${this.#onSelect}></slot>
+    `;
+  }
+
+  /** sets focus on active tile */
+  focus() {
+    (this.#tabindex?.activeItem || this.#tabindex.firstItem)?.focus();
+  }
+
+  /**
+   * programatically select a tile
+   * @param tile {RhTile | null | undefined} tile to select
+   */
+  selectTile(tile: RhTile | null | undefined) {
```

Comment:
we should probably settle on a nomenclature and call all of these `selectItem` etc, aligning names around RTIController in all element

Reply:
to clarify this should be cross referenced and changed before release in order to prevent breaking changes